cEP-0029.md

```diff
@@ -0,0 +1,63 @@
+# Gitmate for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 29                                                    |
+| Version  | 1.0                                                   |
+| Title    | Gitmate for coala                                     |
+| Authors  | Sanchit Gupta <mailto:sanchitgupta.072@gmail.com>     |
+| Status   | Implementation Due                                    |
+| Type     | Process                                               |
+
+## Abstract
+
+This cEP describes the improvements to IGitt and Gitmate, which include
+upgrading dependencies, making it functional, changes to plugins, etc. as a part
+of the [GSoC project](https://summerofcode.withgoogle.com/projects/#6263057774280704).
+
+## Introduction
+
+Gitmate, a git workflow management bot created by coala, hasn't been actively maintained in the last 3 years. My project aims to update it and bring it back to active use for the coala community. I plan to undertake various improvements to Gitmate as well as IGitt, the underlying library that powers Gitmate's git operations.
```

Comment:
Change the sentencing here to represent a set of improvements.

Reply:
Also, this is not just for coala community. We will be enabling gitmate for all users.